no-computer.md

```diff
@@ -58,12 +74,17 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+Time would mainly be spent on the language aspects of the project. I think that the main goals would be to design and optimize language commands for efficiently searching through recipes. I don't envision many complicated semantics, especially if the DSL boils down to various operations on a database of recipes.
```

Comment:
I agree that most of the time would probably be spent on the language aspects of the project, especially considering the systems aspect is pretty straight forward (especially compared to the other ideas).